client/src/test/java/org/spine3/base/FailureThrowableShould.java

```diff
@@ -34,34 +38,54 @@
 
 public class FailureThrowableShould {
 
+    private Command command;
+
+    @Before
+    public void setUp() {
+        final TestCommandFactory commandFactory =
+                TestCommandFactory.newInstance(newUuid(),
+                                               ZoneOffset.getDefaultInstance());
+        this.command = commandFactory.create(newStringValue(newUuid()),
```

Comment:
It could be made shorter:

```java
this.command = commandFactory.create(newUuidValue());
```
Here we call the method inherited from `CommandFactory`. The parameter is a utility factory from `Tests`.